paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -645,6 +653,14 @@ def autoscale_marathon_instance(marathon_service_config, marathon_tasks, mesos_t
             line='Staying at %d instances (%s)' % (current_instances, humanize_error(error)),
             level='debug',
         )
+    meteorite_dims = {
```

Comment:
needs 'instance'

Reply:
otherwise metrics for main_uswest1 will get mixed with api_canary, etc

Reply:
actually can you do

instance_name
service_name
paasta_cluster

so that they will be compatible with our existing metrics
https://app.signalfx.com/#/dashboard/DJzYJDkAcAA?density=4&variables%5B%5D=PaaSTA%20Cluster%3Dpaasta_cluster:&variables%5B%5D=Service%3Dservice_name:&variables%5B%5D=Instance%3Dinstance_name: